app/src/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -649,4 +649,7 @@ public Spannable forceMarkdown(String text){
     public Spannable stylize(String text){
         return MarkupUtil.styleSpannable(getContext(), text);
     }
+
+    public void setBinaryData(Object answer) {
+    }
```

Comment:
Removed `IBinaryWidget` interface and moved the `setBinaryData` method here. This lets us not need to check `instanceof IBinaryWidget` in `ODKView.setBinaryData`.